modules/custom/social_ajax_comments/src/Controller/AjaxCommentsController.php

```diff
@@ -181,6 +181,14 @@ public function socialAdd(Request $request, EntityInterface $entity, $field_name
       // AjaxCommentsForm::save() during the previous HTTP request.
       $cid = $this->tempStore->getCid();
 
+      // Reset cache of the commented entity.
```

Comment:
Do we know why this is needed? Inline documentation should explain it better here.

Do we have a plan or follow-up for this to do? If not, I would request you to please debug and add a proper solution here.

Reply:
@ksis Could you check @navneet0693 comment, please?
cc @ribel

Reply:
@navneet0693 @Ressinel We discussed this case internally with our whole team.
Unfortunately we already spend few days on fixing this one bug and we don't have ability to continue work on it, so we didn't create a follow up. Also "add a proper solution here" is a bit vague suggestion, so we can't estimate and plan it. 
 
If adding better description to the ToDo will unblock this issue we can add it.
But if not - then we can leave it as is.
If Open Social core team can continue work on it that would be great.
If not - then you can close the PR. 
cc @tbsiqueira @bramtenhove

Reply:
@ribel I agree, my suggestion is a bit vague. I will try to investigate this in the coming days and come up with a proper suggestion or fix. If this is blocking you/the team, please feel free to merge it.

Reply:
Thanks @navneet0693. it's not blocking our team, and since this is a small change in code we an use it as a patch.
But would be great if you can investigate it when you have a time.

Reply:
@ribel Sure, Taras. I will take over this PR.